CommunityFridgeMapApi/dependencies/python/db.py

```diff
@@ -51,13 +70,57 @@ class DB_Item:
     REQUIRED_FIELDS = []
     ITEM_TYPES = {}
     TABLE_NAME = ""
+    FIELD_VALIDATION = {}
 
     def __init__(self, db_client: "botocore.client.DynamoDB"):
         self.db_client = db_client
         pass
 
-    def add_item(self):
-        pass
+    def add_item(self, conditional_expression=None) -> DB_Response:
+        """
+        adds item to database
+            Parameters:
```

Comment:
nit: why the indent for Parameters?

Reply:
I was just following the format that this website had: https://www.programiz.com/python-programming/docstrings under `2. Docstrings for Python Functions`

Is there a format you prefer? I know there are a couple different formats we can follow for docstring

Reply:
Not really, just felt that the extra indent was slightly weird. But it's a valid format as mentioned, so that's a good idea.